### PR TITLE
Bug #71: Withdraw fixes

### LINKED ISSUE #71: withdraw is not working

### PR DESCRIPTION
**Pull request by andre-meyer**
# Description

Closes #71 

The changes from the scripts team needed to be implemented in a different way which is done in this PR.

# To Test
- [ ] Create a new strategy
- [ ] Start the withdraw request
- [ ] Observe the 5min wait time
- [ ] Withdraw claim the funds

# Background
/

### CODE REVIEW

**Comment by andre-meyer** [comment]
Please review @alfetopito (I will merge now but merge-back changes later)

File: src/api/withdrawFunds.ts [hunk 1]
```diff
@@ -1,73 +1,92 @@
...
-    }, true
+
+  const  amountFunction = async function (bracketAddress, tokenData, exchange) : Promise<String> {
+    const amount = (await exchange.getBalance(bracketAddress, tokenData.address)).toString()
+    const usdValue = await amountUSDValue(amount, tokenData, {})
+    logger.log(`(request) ${bracketAddress} holds ${amount} (${usdValue}$) in ${tokenData.symbol}`);
+    if (usdValue.gte(ONE)) {
+      return MAXUINT256.toString()
+    } else {
+      return "0"
```
Comment (alfetopito) [inline src/api/withdrawFunds.ts:33]: I know this is something from the script and nothing new to the CMM app, but what about tokens that do not have yet a correct price estimation? 
Won't they have the option to bypass this check and force withdrawing anyway?

File: src/routes/Strategies/Active.js [hunk 2]
```diff
@@ -159,18 +159,20 @@ const Active = () => {
         "LOADING"
       );
       try {
-        const { txs } = await withdrawRequest(context, strategy);
-        if (txs.length === 0) {
+        const txs = await withdrawRequest(context, strategy);
+        if (txs == null || txs.length === 0) {
```
Comment (alfetopito) [inline src/routes/Strategies/Active.js:163]: Same as `!txs` ?

File: src/api/withdrawFunds.ts [hunk 3]
```diff
@@ -1,73 +1,92 @@
...
   ]);
-  
-  const withdrawHelpers = runInitializerIfNotRan(context);
 
-  const argv = {
-    masterSafe: context.safeInfo.safeAddress,
-    brackets: strategy.brackets.map(bracket => bracket.address),
-    tokens: [strategy.baseTokenAddress, strategy.quoteTokenAddress],
-  };
+  const  amountFunction = async function (bracketAddress, tokenData, exchange) : Promise<String|BN> {
```
Comment (alfetopito) [inline src/api/withdrawFunds.ts:67]: I think it should be lower case `string` on `Promise<String|BN>`

**Comment by elena-zh** [comment]
Withdraw is not working on the 189412279 - 98816d8fae540d8fa2633bcbc0741dcadd62c4d0 app version. 
See the comment https://github.com/gnosis/safe-cmm-app-react/issues/71#issuecomment-706956757

**Comment by elena-zh** [comment]
I can withdraw funds in cases when USDc token is not present in a strategy - reported in #87
Move the task to Done, however, the feature is not stable enough: see additional issues #109 and #113

**Pull request merged**